gcc/rust/hir/rust-hir-dump.cc

```diff
@@ -467,11 +603,44 @@ void
 Dump::visit (EmptyStmt &)
 {}
 void
-Dump::visit (LetStmt &)
-{}
+Dump::visit (LetStmt &let_stmt)
+{
+  indent++;
+  // TODO: outer attributes
+  stream << std::string (indent, indent_char);
+  stream << "let ";
```

Comment:
Probably clearer to have something like `LetStmt: ` like we have for `Stmt: `, `Expr: `, `Function: ` etc